core/frontend/src/BriefcaseConnection.ts

```diff
@@ -20,10 +20,16 @@ import { ProgressCallback } from "./request/Request";
 import { disposeTileTreesForGeometricModels } from "./tile/internal";
 import { Viewport } from "./Viewport";
 
+/**
+ * Called to show progress during a download.
+ * @public
+ */
+export type ProgressFunction = (progress: { loaded: number, total: number }) => void;
```

Comment:
Why not call this `DownloadProgressCallback` since that's what it's for and then you can tell people what the units of `loaded` and `total` are.

Reply:
Technically, it could be used for something else as well (e.g. upload).

Reply:
If you think that's likely, perhaps `TransferProgressCallback`.
I am trying to 1) make it obvious that this is a callback that reacts to progress, not a function that produces progress and 2) make it clear to the user what these `loaded` and `total` numbers actually represent.

Reply:
though frankly types are free. Why not just have another one for upload if that comes up? Calling this `OnDownloadProgress` and leaving it `loaded` seems fine to me.

Reply:
Renamed it to `OnDownloadProgress`.